### DISCUSSION TITLE
custom drop down in edit/save

### DISCUSSION THREAD

**Discussion by Shauryadhaka** [aminalaee/sqladmin#725] [Q&A] [answered]
Hey, is there are any way to add a dropdown that will display elements that i can fetch via an api

**Comment by aminalaee** [comment]
Hi, yeah sure, you need to override the form widgets

**Reply by Shauryadhaka** [reply]
Hey, 
Not able to understand that. Can you point me to some documentation that can help here?

**Reply by lukeclimen** [reply, marked as answer]
https://aminalaee.dev/sqladmin/api_reference/model_view/#sqladmin.models.ModelView.form_widget_args
